docs/src/instance_manager.md

```diff
@@ -1,3 +1,9 @@
+---
+id: instance_manager
+sidebar_position: 110
+title: Instance Manager
```

Comment:
This will change the title from Postgres instance manager to Instance Manager, not so explicit